packages/soql-builder-ui/src/modules/querybuilder/app/app.html

```diff
@@ -66,7 +66,9 @@
               <template if:true={showUnsupportedNotification}>
                 <!-- TODO: i18n -->
                 <strong>Your query contains statements that SOQL Builder doesn't currently
-                support.</strong><p>You can use a a text editor to edit the query. If you
+                support.</strong>
+                <p>You can use the 'Run Query' button to run your query as is.</p>
+                <p>You can use a a text editor to edit the query. If you
```

Comment:
```suggestion
                <p>You can use a text editor to edit the query. If you
```
There is a typo here with 2 `a`s

Reply:
Maybe we want to say
```
You can also use a text editor...
```
Adding, "also", I think this will join the two statements better about their options.

Reply:
Good catch! that double "a" has been there for awhile and nobody caught it (not even Emily or Sonal) :-).

Regarding adding "also",  Sonal proposed changing the phrase to

     Consider using a text editor to edit the query

which: _"... I think that might make the “Edit Query Anyway” button more valid..."_.  So I'll do that unless somebody objects.